src/base1/cockpit.js

```diff
@@ -1577,8 +1577,13 @@ function factory() {
      * Use application to prefix data stored in browser storage
      * with helpers for compatibility.
      */
-    function StorageHelper(storage) {
+    function StorageHelper(storageName) {
         var self = this;
+        var storage;
+
+        try {
+            storage = window[storageName];
+        } catch (e) { }
```

Comment:
We really need to shim out the entire thing. Worked on this with @muelli. Will talk about this more today. But this is a first step, and we need to decide what to do as a follow-up next.